entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+source $NVM_DIR/nvm.sh
+nvm use ${NODE_VERSION}
+
+if [[ "${BASE_DIR}" == "" ]]; then
+    BASE_DIR=.
+fi
+
+export DEBUG_LEVEL=3
+export DEBUG="exachimp:*"
+export ROOT_URL="http://127.0.0.1:3001"
```

Comment:
I forgot - what is chimp using `ROOT_URL` for?

Reply:
Oh, OK - I see, the app is using it and you are repurposing the entrypoint script.

Reply:
We could consider adding a bit of explanation in the file as a docstring / inline comment, perhaps.

Reply:
It actually wasn't clear to me that chimp required it, I was just trying to mimic as carefully as possible the logic in `run-tests.sh`. If it is used by chimp to find the running application, why would we need to also provide `--serverHost` and `--serverPort`?

Reply:
Chimp is not using ROOT_URL, the application does - I did not understand that this entrypoint is reused for both when asked above. I added inline comments in https://github.com/Exabyte-io/materials-designer/pull/42/commits/82d5f5e950fd6d6c035e6387b8447225f953deb1